iceoryx_posh/source/roudi/port_manager.cpp

```diff
@@ -279,23 +450,24 @@ void PortManager::handleInterfaces() noexcept
                 }
             }
         }
-        // also forward services from service registry
-        /// @todo #415 do we still need this? yes but return a copy here to be stored in shared memory via new
-        /// StatusPort's
-        /// @todo iox-#27 I guess this was necessary since a service could be offered via ServiceDiscovery;
-        /// this was removed and I somehow have the feeling this breaks the interface ports with the changes from this
-        /// PR if the CaproServiceType is something different than NON
```

Comment:
I think it should be fine to remove this. We could have services without publishers that were offered via the runtime. This was to support ara::com services with methods only that were covered by another communication channel. I think with the servers now it is fine to forward the server offers and there are no more services.